card/lib/card/model/save_helper.rb

```diff
@@ -231,21 +235,23 @@ def add_coderule_item name, prefix, type_id, to
 
 
       def method_missing method, *args
-        return super unless (cardtype_card = extract_cardtype_from_method_name method)
-        Card.create! create_args(*args).merge(type_id: cardtype_card.id)
+        method_name, cardtype_card = extract_cardtype_from_method_name method
+        return super unless method_name
+        args = standardize_args(*args)
+        send "#{method_name}_card", args.merge(type_id: cardtype_card.id)
       end
 
       def respond_to_missing? method, _include_private=false
         extract_cardtype_from_method_name(method) || super
       end
 
       def extract_cardtype_from_method_name method
-        return unless method =~ /^create_(?<type>.+)$/
+        return unless method =~ /^(?<method_name>create|ensure)_(?<type>.+)(?:_card)?$/
```

Comment:
haven't seen these.  var names in regex.  I like.

Reply:
Then you missed several chances in recent years to get to know each other :smile: The last time you passed each other without notice was probably here https://github.com/decko-commons/decko/pull/80/files#diff-be05844104fae4d5b9ea9e1b41dd8040R3